sources/vega/events/v1/events.proto

```diff
@@ -246,6 +246,8 @@ message ValidatorUpdate {
   string name = 7;
   // AvatarURL of the validator
   string avatar_url = 8;
+  // Number of Vega public key
```

Comment:
No sure the comment reflect what this is. Is this the pubkey index or the total number of the pubkey the validator has used so far, or ?

Reply:
I'd go with "Vega public key derviation index"?

Reply:
Minor: Ah shoot, looks like I spelt "derivation" wrong above and you've copied it in, sorry! Can you correct it?